### ISSUE TITLE
needs a dispose

### ISSUE THREAD

**Issue by ryanflorence** [vvo/in-viewport#7]
```js
var viewportListener = inViewport(el, cb);
// later
viewportListener.dispose();
```

Otherwise long-running apps can't use this w/o a memory leak.

**Comment by vvo** [comment]
I agree, would you mind doing a PR? in-viewport is fully tested and you can TDD on it.

I do not have a lot of time right now to tackle this.

@ryanflorence Where are you using in-viewport? Thanks.

**Comment by vvo** [comment]
see #7

**Comment by tzi** [comment]
More likely, see #10 ;)

**Comment by vvo** [comment]
fixed with 3.4.0
